settings.py

```diff
@@ -10,6 +10,7 @@
 	'Activity': 'auction',
 	'AuctionOfLot': 'auction_of_lot',
 	'Bidding': 'bidding',
+	'Drawing': 'drawing',
```

Comment:
Does it need its own model? Can't it be auction_of_lot?

Reply:
Didn't know if we wanted it the same as auction of lot, or if there might be updates in the future that would benefit from a separate model. Right now it's mostly the same, but I think that's just because the lottery data we have is pretty sparse.

Reply:
Want me to update it to be auction_of_lot?